sdk/keyvault/azure-keyvault-administration/tests/test_backup_client.py

```diff
@@ -47,11 +55,14 @@ def test_full_backup_and_restore(self, container_uri, sas_token):
         backup_client = KeyVaultBackupClient(self.managed_hsm["url"], self.credential)
         backup_poller = backup_client.begin_backup(container_uri, sas_token)
 
+        token = backup_poller._polling_method.get_continuation_token()
```

Comment:
I played around with the debugger here and saw some odd behavior:
```py
>>> backup_poller.done()
True
>>> backup_poller.status()
'InProgress'
>>> backup_poller.polling_method().finished()
False
>>> backup_poller.polling_method().status()
'InProgress'
>>> backup_poller.polling_method().update_status()
...
AttributeError: 'PipelineResponse' object has no attribute 'split'
```
Which made me sweat a little because my biggest anxiety about this change is that the code will look correct to me and function as expected on the golden path but break easily otherwise. So, I think thorough test coverage is crucial.